src/Essentials/src/Geolocation/Geolocation.android.cs

```diff
@@ -365,8 +380,11 @@ void ILocationListener.OnProviderEnabled(string provider)
 				activeProviders.Add(provider);
 		}
 
-		void ILocationListener.OnStatusChanged(string provider, Availability status, Bundle extras)
+		void ILocationListener.OnStatusChanged(string? provider, Availability status, Bundle? extras)
 		{
+			if (provider == null)
```

Comment:
as ILocationListener has the `provider` parameter marked with string?, provider may theoretically be null, but from reading the Android documentation, it makes no sense to be null. Therefore I just return here.

Reply:
@jfversluis the second...